templates/serverpod_templates/projectname_server/web/templates/default.html

```diff
@@ -4,6 +4,7 @@
     <meta charset="utf-8">
     <title>Built with Serverpod</title>
     <link rel="stylesheet" href="/css/style.css">
+    <link rel="icon" type="image/x-icon" href="https://serverpod.dev/assets/img/favicon/favicon.ico">
```

Comment:
This should not reference an external icon, but the icon should be in the static folder.